### ISSUE TITLE
`DenseMap`: `Get` takes forever after a few emplacing and removing

### ISSUE THREAD

**Issue by Jason5Lee** [Wsm2110/Faster.Map#39]
```csharp
var dense = new DenseMap<int, int>();
for (int i = 1; i <= 16; ++i)
{
    dense.Emplace(i, 0);
    dense.Remove(i);
}

dense.Get(0, out int value); // Takes forever 
```

**Comment by Wsm2110** [comment]
Thnx mate,

Ill look into it when i get home

**Comment by Jason5Lee** [comment]
> Thnx mate,
> 
> Seems to work just fine. What version are you using?
> 
>       [Fact]
>       public void Get_ShouldReturn_WhileMapIsEmpty()
>       {
>           var dense = new DenseMap<int, int>();
>           for (int i = 1; i <= 16; ++i)
>           {
>               dense.Emplace(i, 0);
>               dense.Remove(i);
>           }
> 
>           var result = dense.Get(0, out int value);
> 
>           Assert.False(result);
>       }
> 
> 

6.0.4

https://dotnetfiddle.net/i1Apb8

**Comment by Wsm2110** [comment]
I guess this is a clever way to dodge the resize behavior. But than again this is already fixed in 6.0.5. I'll release it when i get home.

**Comment by Wsm2110** [comment]
Thank you for your time reporting this, it's appreciated.

Have fun with 6.1.0
